integration/mediation-tests/tests-other/src/test/java/org/wso2/carbon/esb/ServerStartupTestCase.java

```diff
@@ -39,7 +39,7 @@ public void initialize() {
     public void testVerifyLogs() {
         carbonLogReader.start();
         String logs = carbonLogReader.getLogs();
-        boolean status = logs.indexOf("Mgt Console URL") < logs.indexOf("Starting WSO2 Carbon");
+        boolean status = logs.indexOf("Mgt Console URL") < logs.indexOf("Starting WSO2 Carbon") || logs.contains("ERROR");
```

Comment:
Do we ever get this "Mgt Console URL" log in Micro Integrator.
Also as we are just starting carbon log reader and getting the logs of a started server, will it get other logs if we have any, If needed we can do a restart and see the server.  .